openapi.yml

```diff
@@ -3384,6 +3387,96 @@ paths:
+                    signins_count: 189
+                    invites_expected_next_hour: 1
+                    invites_by_hour:
+                      - range_start: '2020-07-17T00:59:59.999Z'
+                        range_end: '2020-07-17T01:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T01:59:59.999Z'
+                        range_end: '2020-07-17T02:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T02:59:59.999Z'
+                        range_end: '2020-07-17T03:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T03:59:59.999Z'
+                        range_end: '2020-07-17T04:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T04:59:59.999Z'
+                        range_end: '2020-07-17T05:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T05:59:59.999Z'
+                        range_end: '2020-07-17T06:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T06:59:59.999Z'
+                        range_end: '2020-07-17T07:59:59.999Z'
+                        expected_visitors: 0
+                      - range_start: '2020-07-17T07:59:59.999Z'
+                        range_end: '2020-07-17T08:59:59.999Z'
+                        expected_visitors: 1
+        '400':
+          description: A generic error
+          content:
+            application/json:
+              schema:
+                $ref: ./models/ErrorsList.v1.yaml
+              examples:
+                default:
+                  value:
+                    errors:
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+                      - domain: some text
+                        attribute: some text
+                        code: some text
+                        message: some text
+        '401':
+          description: "You don't have permission to view the details of the location's capacity."
+        '403':
+          description: You do not have permission for this action
+      operationId: getLocationCapacity
+      description: Gets the details of the capacity in a location.
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: local_time
+          description: Current local time with offset value.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: ./models/Capacity.v1.yaml
+            examples:
+              default:
+                value:
+                  timestamp: '2020-07-16T17:05:08-07:00'
+                  timezone: America/Vancouver
+                  hours_to_calculate: 8
```

Comment:
A `get` can't have a request body at all.

Reply:
removed it